phpBB/styles/prosilver/template/smiley_dropdown.html

```diff
@@ -0,0 +1,41 @@
+<div id="smiley-dropdown" class="dropdown dropdown-extended" >
+    <div class="pointer"><div class="pointer-inner"></div></div>
+    <div class="smiley-content">
+        <div id="smiley-box" class="smiley-box">
+            <ul>
+            <!-- EVENT posting_editor_smilies_before -->
+            <!-- IF S_SMILIES_ALLOWED and .smiley -->
+
+            <!-- BEGIN smiley -->
+                <li class="smiley-list">
+            <a href="#" onclick="insert_text('{smiley.A_SMILEY_CODE}', true); return false;"><img src="{smiley.SMILEY_IMG}" width="{smiley.SMILEY_WIDTH}" height="{smiley.SMILEY_HEIGHT}" alt="{smiley.SMILEY_CODE}" title="{smiley.SMILEY_DESC}" /></a>
```

Comment:
This was used inside the code. 
`	<!-- IF S_SMILIES_ALLOWED and .smiley -->
			<strong>{L_SMILIES}</strong><br />
			<!-- BEGIN smiley -->
				<a href="#" onclick="insert_text('{smiley.A_SMILEY_CODE}', true); return false;"><img src="{smiley.SMILEY_IMG}" width="{smiley.SMILEY_WIDTH}" height="{smiley.SMILEY_HEIGHT}" alt="{smiley.SMILEY_CODE}" title="{smiley.SMILEY_DESC}" /></a>
			<!-- END smiley -->
		<!-- ENDIF -->`

That's why I used this.. Do I need to insert this function to forum_fn.js file.